js/piwik.js

```diff
@@ -7465,7 +7465,8 @@ if (typeof window.Matomo !== 'object') {
                     maq_optout_by_default: tracker.isConsentRequired()
                 });
             }
-        }, false);
+        }, fal
+        se);
```

Comment:
Guess you added this line break by accident:

```suggestion
        }, false);
```